src/Contact.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Created by brianmichael on 1/23/17.
+ */
+import React from 'react';
+import Highlighter from './Highlighter';
+
+const Contact = (props) => {
+
+  return (
+    <li
+      className="contact"
+      >
+      <div className="image-cropper">
+        <img src={props.avatar} alt="avatar" />
+      </div>
+      <div className="contact-info">
+        <h2>
+          <Highlighter searchWords={props.value} textToHighlight={props.name} />
+        </h2>
+        <h3>{props.occupation}</h3>
+      </div>
+
+      <button
+        className="select-button"
+        id={props.id}
+        onClick={(event) => props.handleSelectContactClick(event)}>Select
```

Comment:
If handleSelectContactClick is not defined as `isRequired` you need to check if it is undefined, otherwise this will break. See comment below.